packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -37,9 +37,63 @@ class ImagePickerPlugin extends ImagePickerPlatform {
     double maxHeight,
     int imageQuality,
     CameraDevice preferredCameraDevice = CameraDevice.rear,
-  }) {
+  }) async {
     String capture = computeCaptureAttribute(source, preferredCameraDevice);
-    return pickFile(accept: _kAcceptImageMimeType, capture: capture);
+    return pickFile(
+      accept: _kAcceptImageMimeType,
+      capture: await _resizeImage(
+        capture,
+        maxWidth.toInt(),
+        maxHeight.toInt(),
+        imageQuality,
+      ),
+    );
```

Comment:
I don't think this'll work. A couple of comments:

* Only apply the resizing if the maxWidth and maxHeight (and possibly imageQuality) are passed in (otherwise do we end up with 0x0 images maybe?)
* This is replacing the value of the "capture" attribute, which is not what we want. Capture does [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#htmlattrdefcapture).

I think the correct flow for this would be to let `pickFile` do its thing, then you'd have a `src` to pass to your _resizeImage method (the `path` of the returned `PickedFile`).

Reply:
OK I understand the logic here and what is doing what.
Thank you very much for the explanation and how to procede here :)
This change has now been commited.